api/modules/city/views.py

```diff
@@ -51,7 +51,7 @@ def get_city(request, city_id):
     try:
         city_visit_log = CityVisitLog(city=city, user=request.user)
         city_visit_log.save()
-    except Exception as e:
+    except Exception:
```

Comment:
Please do not add files which are not relevant to the fix.
If you want to remove these unused alias, you are raise another PR

Reply:
@bjornwelboren Please remove this file from commit as there is no relevant change here.
Do the same for the other 2 files also.